lib/core/scheduler.rb

```diff
@@ -74,14 +74,14 @@ def sync_triggered?(connector_settings)
 
       # Sync when sync_now flag is true for the connector
       if connector_settings[:sync_now] == true
-        Utility::Logger.info("#{connector_settings.formatted.capitalize} is manually triggered to sync now.")
+        Utility::Logger.debug("#{connector_settings.formatted.capitalize} is manually triggered to sync now.")
```

Comment:
Does this print out a lot? I would think this might still be valuable as `info` if it's only logged once when something is manually triggered.

Reply:
Not for Crawler. Agree with your point